include/open62541pp/types/DataValue.h

```diff
@@ -52,16 +52,16 @@ class DataValue : public TypeWrapper<UA_DataValue, UA_TYPES_DATAVALUE> {
 
     /// Create DataValue from scalar value.
     /// @see Variant::fromScalar
-    template <typename... Args>
+    template <VariantPolicy Policy = VariantPolicy::ReferenceIfPossible, typename... Args>
```

Comment:
The default policy here should also be Copy, correct?

Reply:
Thanks! Fixed in https://github.com/open62541pp/open62541pp/pull/174/commits/6adb6ca553b89a0d0d758a48e2ad9b0d463488cc